docs/admin/cluster-management.md

```diff
@@ -66,37 +66,56 @@ Instance Group will take care of putting appropriate image on new machines and s
-the autoscaler will choose the bigger number. The number of nodes in the cluster set by the autoscaler will be limited from `KUBE_AUTOSCALER_MIN_NODES` (default: 1)
-to `KUBE_AUTOSCALER_MAX_NODES` (default: the initial number of nodes in the cluster).
+Cluster autoscaler also scales down the cluster if it notices that some node is not needed anymore for 
+an extended period of time (10min but it may change in the future) in a sence that:
 
-The autoscaler is implemented as a Compute Engine Autoscaler.
-The initial values of the autoscaler parameters set by `kube-up.sh` and some more advanced options can be tweaked on
-`Compute > Compute Engine > Instance groups > your group > Edit group`[Google Cloud Console page](https://console.developers.google.com)
-or using gcloud CLI:
+* It has more that 50% of cpu and memory not requested by any pod.
+* All of its pods can be easily restarted on the other nodes (they are replciated, don't have local storage) and there 
+  is no important system pods among them.
+
+Cluster autoscaler is configured per instance group (GCE) or node pool (GKE). 
+
+If you are using GCE then you can either enable it while creating a cluster with kube-up.sh script. 
+To configure cluser autoscaler you have to set 3 environment variables:
+
+* `KUBE_ENABLE_CLUSTER_AUTOSCALER` - it enables cluster autoscaler if set to true.
+* `KUBE_AUTOSCALING_MIN_NODES` - minimum number of nodes in the cluster.
+* `KUBE_AUTOSCALING_MAX_NODES` - maximum number of nodes in the cluster.
+
+Example:
+
+```shell
+KUBE_ENABLE_CLUSTER_AUTOSCALER=true KUBE_AUTOSCALING_MIN_NODES=3 KUBE_AUTOSCALING_MAX_NODES=10 NUM_NODES=5 ./cluster/kube-up.sh
+```
+
+On GKE you configure cluster autoscaler either on cluster creation or update or when creating a particular node pool
+(which you want to be autoscaled) by passing flags `--enable-autoscaling` `--min-nodes` and `--max-nodes`
+to the corresponding `gcloud` commands.
+
+Examples:
+```shell
+gcloud container clusters create mytestcluster --zone=us-central1-b --enable-autoscaling=true --min-nodes=3 --max-nodes=10 --num-nodes=5
+```
 
 ```shell
-gcloud alpha compute autoscaler --zone $ZONE <command>
+gcloud container clusters update mytestcluster --enable-autoscaling=true --min-nodes=1 --max-nodes=15
 ```
 
-Note that autoscaling will work properly only if node metrics are accessible in Google Cloud Monitoring.
-To make the metrics accessible, you need to create your cluster with `KUBE_ENABLE_CLUSTER_MONITORING`
-equal to `google` or `googleinfluxdb` (`googleinfluxdb` is the default value). Please also make sure
-that you have Google Cloud Monitoring API enabled in Google Developer Console.
+**Cluster autoscaler assumes that all nodes inside a signle instance group or node pool are roughly the same, they don't have any manually added labels, etc. Please create a separate instance group/node pool if you want to add some labels to them.**
```

Comment:
s/signle/single

I'd expect that user will the same concerns regarding "roughly the same". If this is only about label we should say this explicitly. Or we can phrase it differently, like "we expect that nodes where not manually modified (e.g. by adding labels via kubectl) as those properties would be propagated to the new nodes within the same instance group".